REWARD_MANAGEMENT.md

```diff
@@ -19,3 +19,13 @@ Fees are calculated and distributed on Home network. Validators will receive nat
 ### Home to Foreign transfer
 Fees are calculated and distributed on Home network. Validators will receive native coins.
 ![erc-native-hometoforeign](https://user-images.githubusercontent.com/4614574/51607508-96f47480-1ef3-11e9-93a1-0f1111793f2a.png)
+
+## ERC-TO-ERC
+
+### Foreign to Home transfer
+Fees are calculated and distributed on Home network. Validators will receive ERC20 tokens.
+![ERC-ERC-ForeignToHome](https://user-images.githubusercontent.com/4614574/55352936-70752b80-5498-11e9-98ec-f2ca4d9fcb8b.png)
```

Comment:
Just noticed that on the pictures for the POSDAO fee manager it is necessary to mention in the scheme that the fee distribution is delegated to the the reward contract registered as part of the network consensus.

Reply:
Thanks! Added the reward contract on the pictures https://github.com/poanetwork/poa-bridge-contracts/pull/166/commits/1ca930ac1db0ebe69e6c1e3652504243fdaa9522

Reply:
Thanks! It is better now but I would modify the schemes:
![image](https://user-images.githubusercontent.com/20793260/56430451-10231e00-62cf-11e9-8bc6-96c750cd0110.png)
"fee tokens are minted and distributed between the network validators"

the same is for the _Home to Foreign transfer_ scheme.

Reply:
Thanks for suggestion. Fixed